### ISSUE TITLE
aria-sort status is not getting announced in Edge

### ISSUE THREAD

**Issue by capozzic1** [FreedomScientific/standards-support#726]
### Summary

 Brief description of the issue. Please list any specific steps.
JAWS is not announcing the status of the aria sort attribute when enter is pressed in Microsoft Edge. 
 
Example:
 1. Go to https://www.w3.org/WAI/ARIA/apg/patterns/table/examples/sortable-table/
 2. Navigate to the table with the caption that says 'Students currently enrolled in WAI-ARIA 101'
 3. Navigate to first name 
 4. Press enter 

### Expected result

JAWS should announce 'enter ascending' or 'enter descending' depending on the sort status. 

### Actual result

JAWS only announces 'enter' 

#### JAWS version and build number
JAWS 2021.2107.12 

 
#### Operating System and version
Windows 10 Enterprise

#### Browser and version:
Microsoft Edge 113.01774.50 64 bit

**Comment by JAWS-test** [comment]
Problem also occurs with JAWS 2022 and 2023.

Problem does not occur with Firefox and Chrome.

Problem is not so severe because the sort direction is noticeable when navigating to the column heading again

**Comment by stevefaulkner** [comment]
Could not reproduce:
on pressing enter the sort was changed and announced
Tested using https://cdpn.io/pen/debug/XWyzzxM, JAWS 2023.2306.38 & Edge Version 114.0.1823.67 (Official build) (64-bit)
